lib/new-admin/admin-server.js

```diff
@@ -66,35 +65,7 @@ app.use(cors({ credentials: true, origin: devMode && 'https://localhost:3001' })
 
 app.use('/id-card-photo', serveStatic(idPhotoCardBasedir, { index: false }))
 app.use('/front-camera-photo', serveStatic(frontCameraBasedir, { index: false }))
-
-app.get('/api/register', (req, res, next) => {
-  const otp = req.query.otp
-  const ua = req.headers['user-agent']
-  const ip = req.ip
-
-  if (!otp) return next()
-
-  return login.register(otp, ua, ip)
-    .then(r => {
-      if (r.expired) return res.status(401).send('OTP expired, generate new registration link')
-
-      // Maybe user is using old registration key, attempt to authenticate
-      if (!r.success) return next()
-
-      const cookieOpts = {
-        httpOnly: true,
-        secure: true,
-        domain: hostname,
-        sameSite: true,
-        expires: NEVER
-      }
-
-      const token = r.token
-      req.token = token
-      res.cookie('token', token, cookieOpts)
-      res.sendStatus(200)
-    })
-})
+app.use('/', register)
```

Comment:
I think this will bring us problems in the future like it is right now because if we want to add a different route like /newapi/register we will have to add it inside the './routes/authentication' file and soon we will run into collision problems. 
In my opinion this should become
`app.use('/api', register)`
and inside the './routes/authentication' file it becomes
`router.get('/register', register)`

Reply:
That's a good point! Changed.